.github/workflows/publish-common.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish reusable workflow
+
+on:
+  workflow_call:
+    inputs:
+      with-update-package-lock:
+        type: boolean
+        default: false
+      cmd:
```

Comment:
**[Note]:**

В случае с выпуском **canary** версии будет использоваться флаг `npx lerna bootstrap --since=$(git merge-base --fork-point origin/dev)`. 

Для выпуска **RC**, **Latest** версии будет выполняться - `npx lerna bootstrap`.